src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -189,8 +189,31 @@ struct Size : tt::ConformsTo<protocols::ControlError> {
         "TimescaleTuner for smoothing horizon measurements."};
   };
 
+  struct MaxAllowedRadialDistance {
```

Comment:
How about we add these three options to their own struct `State5Options` so it's easier to disable? Doing this

```cpp
  struct State5Options {
    struct MaxAllowedRadialDistance {}
    struct OutwardDriftVelocity {}
    struct OutwardDriftTimescale {}
    // rest of the option stuff and member variables
    
    using type = Options::Auto<State5Options, Options::AutoLabel::None>;
    static constexpr Options::String help = ...;
  };

  using options = tmpl::list<..., State5Options, ...>;

  Size(..., const std::optional<State5Options>& state_5_options, ...);
```

lets us do in the input file

```yaml
ControlSystems:
  Size:
    ControlError:
      ...
      State5Options: None
```

in order to disable state 5. I think this is clearer for users than having to specify 0 velocity or a super large timescale in order to disable it. This might mean a few other `double`s in this change have to now be `std::optional<double>`